source/PreferencesPanel.cpp

```diff
@@ -86,6 +86,18 @@ namespace {
 	const int SETTINGS_PAGE_COUNT = 2;
 	// Hovering a preference for this many frames activates the tooltip.
 	const int HOVER_TIME = 60;
+
+	const map<string, SoundCategory> volume_bars = {
+			{"volume", SoundCategory::MASTER},
+			{"master volume", SoundCategory::MASTER},
+			{"music volume", SoundCategory::MUSIC},
+			{"ui volume", SoundCategory::UI},
+			{"weapon volume", SoundCategory::WEAPON},
+			{"engine volume", SoundCategory::ENGINE},
+			{"jump volume", SoundCategory::JUMP},
+			{"explosion volume", SoundCategory::EXPLOSION},
+			{"environment volume", SoundCategory::ENVIRONMENT}
+	};
```

Comment:
Could all these similar maps be merged together?

Reply:
Well, the PreferencesPanel map has every value once, this has MASTER twice, and the Effect map doesn't have MASTER and MUSIC, so it wouldn't be trivial.